tests/isotope_qty_test.py

```diff
@@ -244,10 +256,19 @@ def test_isotopequantity_time_when_error(stable_isotope):
 def test_isotopequantity_activity_now(iq):
     """Test IsotopeQuantity.*_now()"""
 
-    assert np.isclose(iq.bq_now(), iq.bq_at(datetime.datetime.now()))
-    assert np.isclose(iq.uci_now(), iq.uci_at(datetime.datetime.now()))
-    assert np.isclose(iq.atoms_now(), iq.atoms_at(datetime.datetime.now()))
-    assert np.isclose(iq.g_now(), iq.g_at(datetime.datetime.now()))
+    with pytest.warns(DeprecationWarning):
+        assert np.isclose(iq.bq_now(), iq.bq_at(datetime.datetime.now()))
+    with pytest.warns(DeprecationWarning):
+        assert np.isclose(iq.uci_now(), iq.uci_at(datetime.datetime.now()))
+    with pytest.warns(DeprecationWarning):
+        assert np.isclose(iq.atoms_now(), iq.atoms_at(datetime.datetime.now()))
+    with pytest.warns(DeprecationWarning):
+        assert np.isclose(iq.g_now(), iq.g_at(datetime.datetime.now()))
+
+    assert np.isclose(iq.bq_at(), iq.bq_at(datetime.datetime.now()))
```

Comment:
These tests might break on very slow machine. They only work right now because `rtol=1e08` by default for isclose. I think it would be better to define an absolute tolerance as a function of of how much decay we would expect in lets say a second.

```
assert np.isclose(iq.bq_at() - iq.bq_at(datetime.datetime.now()), iq.bq_during(iq.ref_time, iq.ref_time + datetime.timedelta(0,1)))
```
Not very pretty but probably safer. Feel free to change this if you see a better way of implementing it.

Reply:
Ah yeah, I figured this could be the case. I'll think about this this afternoon.

Reply:
The issue with using 1 s is that 1 s can mean very different activities at `now` vs at the `ref_time`

Reply:
We could possibly do a speed test and `@pytest.mark.skipif` if the two `datetime.datetime.now()` are too different. Or we could just loosen the `rtol` in `np.isclose`

Reply:
`rtol` is very generous for large numbers, not sure if it is needed. Maybe it is fine as it is.

Reply:
How about this sequence:
```
x0 = iq.bq_at()
x1 = iq.bq_at(datetime.datetime.now())
x2 = iq.bq_at()
```
and test that `x2 <= x1 <= x0`.

Reply:
done